### ISSUE TITLE
Edit form: Overview section

### ISSUE THREAD

**Issue by paaljoachim** [CalderaWP/Caldera-Forms#875]
Hey

I have a long form and have to click through each field to be sure that all the fields that I wanted included in the entries field are actually included.

Edit form area.
When not clicking any field (no field active). The last field selected will still be seen.
Here is a great area to show additional field information associated with the form being edited.
Such as an overview of entries included in an Entry list.

Something like....

**Form field overview**
Fields included in the entry list:
Name
Email
Message

Additional unique information. Let say you have a custom CSS class connected with one of the fields. 
A placeholder on another field.
Etc

Most often I believe there will not be a lot in the additional unique information area. 

The bottom line is that this area when no field is selected can be used to give the overview of the form. Btw there could be suggestion as to do you want to include Telephone number to the entry list? As the user might have forgotten to include it...:)

Have a great day!
